doc/TBG_macros.cfg

```diff
@@ -188,6 +188,19 @@ TBG_restart="--restart"
 # Note: does currently not work with `Radiation` plugin
 TBG_softRestarts="--softRestarts 5000"
 
+# ISAAC server and rendering interval settings
+TBG_isaac="--isaac.period 1 --isaac.name !TBG_jobName --isaac.url <server_url>"
+
+# ISAAC JPEG quality (1..100). Smaller values are faster sent, but of lower quality
+TBG_isaac_quality="--isaac.quality 80"
+
+# ISAAC render resolution (default 1024x768)
+TBG_isaac_resolution="--isaac.width 1024 --isaac.height 768"
+
+# ISAAC direct pause option telling the ISAAC plugin to pause directly after
+# the start of the simulation
+TBG_isaac_pause="--isaac.direct_pause true"
```

Comment:
The default is `false`. So `--isaac.directPause` does the same as `--isaac.directPause true`?